app/src/main/java/fr/publicissapient/planningpoker/model/Cards.kt

```diff
@@ -0,0 +1,5 @@
+package fr.publicissapient.planningpoker.model
+
+data class Cards(
+	val red: List<Card>
```

Comment:
si cette classe est sensé contenir toutes les cartes de tous les couleurs, on pourrait renommer ce variable en `redCards` plutôt ?

Reply:
It changes later (I added colour, check in others PR)